data/hai/hai reveal 5 philosophers.txt

```diff
@@ -421,6 +427,7 @@ mission "Hai Reveal [B06-long] Tarazed Analysis"
 			choice
 				`	(Return to your ship.)`
 			`	After one last ride through the metro system of <origin>, you arrive at your ship. You plug the data card into your computer, and a new "government" shows up: Smuggler. The ships will be colored differently than pirates.`
+			`	After a moment's thought, you realize that if the smuggling involves Devil's Run, you may find the most smugglers in systems near there.`
```

Comment:
Redundant with earlier direction, and also no.
This is the B-plotline if the player hasn't actually discovered how to access a wormhole like this before - which is normally something that requires engagement/thought and effort.
It is not meant to spoonfeed the player here.

Reply:
Agreed. Will remove.